com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/resources/ConfigurationEntriesResource.java

```diff
@@ -165,9 +165,14 @@ public Response getConfigurationEntry(@PathParam(ID) long id) throws SLException
     @POST
     @Consumes(MediaType.APPLICATION_XML)
     public Response createConfigurationEntry(String xml) throws SLException {
+        
         ConfigurationEntryDto dto = parseDto(xml, CREATE_CONFIGURATION_ENTRY_SCHEMA_LOCATION);
-
         ConfigurationEntry result = entryDao.add(dto.toConfigurationEntry());
+        
```

Comment:
The exception here is thrown after inserting the result in the DAO. Could this potentially lead to a corrupted database content?